tests/lib/tools/invariant-tool

```diff
@@ -74,6 +75,34 @@ check_lxcfs_mounted() {
 	fi
 }
 
+check_stray_dbus_daemon() {
+	n="$1" # invariant name
```

Comment:
Nitpick, could use a more descriptive var name.

Reply:
I followed a pattern shared by other but I actually thinking about it, I can make invariant code easier to handle by removing that whole section and handling it once below. tl;dr; is that shared code, including this name, will go away. I'll send a follow-up.

Reply:
Sounds good, thank you.